docs/features/models/neural_ranking.ipynb

```diff
@@ -0,0 +1,250 @@
+   "source": [
+    "!pip install -q deeppavlov"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Then make sure that all the required packages for the model are installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -m deeppavlov install ranking_ubuntu_v2_torch_bert_uncased"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`ranking_ubuntu_v2_torch_bert_uncased` is the name of the model's *config_file*. [What is a Config File?](http://docs.deeppavlov.ai/en/master/intro/configuration.html)\n",
+    "\n",
+    "There are alternative ways to install the model's packages that do not require executing a separate command -- see the options in the next sections of this page.\n",
+    "The full list of models for neural ranking with their config names can be found in the [table](#3.-Models-list).\n",
+    "\n",
+    "# 3. Models list\n",
+    "\n",
+    "| Config | Language | Dataset | Transformer model |\n",
+    "| :--- | :---: | :--- | :--- |\n",
+    "| ranking/ranking_ubuntu_v2_torch_bert_uncased.json | En | [Ubuntu v2](https://github.com/rkadlec/ubuntu-ranking-dataset-creator) | bert-base-uncased |\n",
+    "| classifiers/paraphraser_rubert.json | Ru | [paraphraser.ru](https://paraphraser.ru) | DeepPavlov/rubert-base-cased |\n",
+    "| classifiers/paraphraser_convers_distilrubert_2L.json | Ru | [paraphraser.ru](https://paraphraser.ru) | DeepPavlov/distilrubert-tiny-cased-conversational |\n",
+    "| classifiers/paraphraser_convers_distilrubert_6L.json | Ru | [paraphraser.ru](https://paraphraser.ru) | DeepPavlov/distilrubert-base-cased-conversational |\n",
+    "\n",
+    "# 4. Use the model for prediction\n",
+    "\n",
+    "## 4.1 Predict using Python\n",
+    "\n",
+    "### English"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from deeppavlov import configs, build_model\n",
+    "\n",
+    "\n",
+    "ranking = build_model(\"ranking_ubuntu_v2_torch_bert_uncased\", download=True, install=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ranking([[\"Forrest Gump is a 1994 American epic comedy-drama film directed by Robert Zemeckis.\",\n",
+    "          \"Robert Zemeckis directed Forrest Gump.\",\n",
+    "          \"Robert Lee Zemeckis was born on May 14, 1952, in Chicago.\"]])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Input:** List[List[sentence1, sentence2, ...]], where the sentences from the second to the last will be ranked by similarity with the first sentence.\n",
+    "\n",
+    "**Output:** List[List[scores]] - similarity scores to the first sentence of the sentences from the second to the last.\n",
+    "\n",
+    "### Russian"
+   ]
+  },
```

Comment:
Thanks!
Are there  plans to fix such as "output is no a code cell", "List[List[scores]]  -> `List[List[scores]]`"?

Reply:
Thanks @IgnatovFedor !
Are there  plans to fix such as "output is no a code cell", "List[List[scores]]  -> `List[List[scores]]`"?